modules/compute/gke-node-pool/main.tf

```diff
@@ -20,8 +20,6 @@ locals {
 }
 
 locals {
-  sa_email = var.service_account_email != null ? var.service_account_email : data.google_compute_default_service_account.default_sa.email
```

Comment:
Are we going to still use the default_service_account if the gke_service_account not provided?
If yes I believe we should keep the current logic and resources.

Reply:
please ignore